frameworks/cassandra/universe/resource.json

```diff
@@ -8,7 +8,8 @@
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
       "scheduler-zip": "{{artifact-dir}}/cassandra-scheduler.zip",
       "cassandra-tar-gz": "https://downloads.mesosphere.com/cassandra/assets/apache-cassandra-3.11.6-bin-dcos.tar.gz",
-      "bootstrap-zip": "{{artifact-dir}}/bootstrap.zip"
+      "bootstrap-zip": "{{artifact-dir}}/bootstrap.zip",
+      "cassandra-exporter-jar": "https://backupstestcassandra.s3-us-west-2.amazonaws.com/cassandra-exporter-agent-0.9.10.jar"
```

Comment:
Hi @viivek46 , can we migrate S3 path of **cassandra-exporter-jar** to official downloads.mesosphere.com.

Reply:
I see you have already made [Comment](https://github.com/mesosphere/dcos-commons/pull/3246#issuecomment-604146950) regarding S3. Since it has passed all CI checks, are we good for the S3 migration?

Reply:
@ankitcid waiting for @alexeygorobets reply before making final changes!